### PR TITLE
{bio}[foss/2016b] MMseqs2 v1-c7a89

### PR DESCRIPTION
**Pull request by pescobar**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/m/MMseqs2/MMseqs2-1-c7a89-foss-2016b.eb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+name = 'MMseqs2'
+version = '1-c7a89'
+
+homepage = 'https://github.com/soedinglab/mmseqs2'
+description = "MMseqs2: ultra fast and sensitive search and clustering suite"
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+
+source_urls = ['https://github.com/soedinglab/MMseqs2/archive/']
+sources = ['%(version)s.tar.gz']
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/m/MMseqs2/MMseqs2-1-c7a89-foss-2016b.eb]: missing checksum

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2421.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/e2b830a8535c4e3777c13be113e398d5 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2057.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/37af5966b9218608523986b6fc704601 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @pescobar!

**Pull request merged**